components/Main/Main.js

```diff
@@ -28,16 +29,18 @@ const Main = () => {
   // });
 
   const UnorderList = document.querySelector('.playerList');
+  const refreshContainer = document.querySelector('.section-refresh_container');
+  const sectionForm = document.querySelector('.section-form');
 
-  const Container = Wrapper({
-    component: 'div',
-    className: 'section-refresh_container',
-  });
+  // const Container = Wrapper({
+  //   component: 'div',
+  //   className: 'section-refresh_container',
+  // });
 
-  const heading1 = PrimaryHeading({
-    textContent: 'Recent scores',
-    className: 'score-heading',
-  });
+  // const heading1 = PrimaryHeading({
+  //   textContent: 'Recent scores',
+  //   className: 'score-heading',
+  // });
```

Comment:
- [ ] Please remove this commented block of code as it makes your work look untidy